### ISSUE TITLE
Incompatibility with Rails 5

### ISSUE THREAD

**Issue by danReynolds** [dejan/rails_panel#119]
I'm trying to update my Rails application to Rails 5 which requires `rack ~> 2.0`, however `meta_request` requires `rack-contrib ~> 1.1` which depends on `rack ~> 1.4`. Does this make the meta_request gem incompatible with Rails 5?

**Comment by christhomson** [comment]
I was able to install `meta_request` on Rails 5 using `rack-contrib 1.2.0`, which relies on `rack >= 0.9.1`.

**Comment by jfragoulis** [comment]
Still the fact remains that the gemspec is wrong. `rack-contrib` seriously need to support rack 2.0, yet `meta_request` need to either drop rails 5.x support or range the `rack-contrib` versions:

```ruby
gem.add_dependency 'rack-contrib', '>= 1.1.0', '<= 1.2.0'
```

PS: There is this weird `1.2.0.39.g17d21b4` version which does not stand right :/

**Comment by axelson** [comment]
FYI, dropping this from our project until this is fixed.

**Comment by modosc** [comment]
i don't have any current projects to test this on but if anyone could try out [this branch](https://github.com/dejan/rails_panel/tree/rails-5-1-support) for me i'd appreciate it. the tests pass but i'd like some more confirmation beyond that.

**Comment by modosc** [comment]
also the only piece of `rack-contrib` we depend on is [response_headers]](https://github.com/rack/rack-contrib/blob/master/lib/rack/contrib/response_headers.rb) which doesn't seem to have changed since 2010

**Comment by jfragoulis** [comment]
@modosc seems to bundle ok for me. Although my upgrade is still in progress, it updates meta_request just fine.

**Comment by modosc** [comment]
ok i merged to master and bumped up the version - @dejan can you roll a release up?

**Comment by dejan** [comment]
Thanks @modosc! New version pushed.

**Comment by aandis** [comment]
Hitting this with the latest version (0.4.3)
```
    meta_request was resolved to 0.4.3, which depends on
      rack-contrib (< 3, >= 1.1) was resolved to 1.5.0, which depends on
        rack (~> 1.4)
```

**Comment by modosc** [comment]
@aandis afaict that's due to [rack-contrib 1.50](https://github.com/rack/rack-contrib/blob/master/rack-contrib.gemspec#L37) - you probably need to lock down your `rack-contrib` or `rack` versions to prevent whatever conflict is occurring.
